components/keys/examples/key_generator.rs

```diff
@@ -13,7 +13,6 @@
 // limitations under the License.
 
 use exonum_keys::*;
```

Comment:
Why is wildcard import not affected? 🙃 Intuitively, it looks more "suspicious" than ones in the test modules, for example.

Reply:
Weird. But clippy skipped it.